### DISCUSSION TITLE
Class to add spacing between elements

### DISCUSSION THREAD

**Discussion by verbose117** [tailwindlabs/tailwindcss#9412] [Ideas]
# Spacing between elements with a new class
I was thinking if **tailwindcss** could add a class to configure the spacing between elements inside a container. This will also allow adding horizontal or vertical or by default for both.

This can be done possible using the direct child selector using css.
```css
> * {
  margin: 1rem;
}

/* horizontal */
> * {
  margin: 0 1rem;
}

/* vertical */
> * {
  margin: 1rem 0;
}
```

This will save much more time adding custom margin between elements inside the container, and can be a very useful feature.

Please let me know, how this feature sounds. If yes, I am ready to work on it 👍 !

**Comment by wongjn** [comment]
I believe this feature is already covered by the [Space Between utilities](https://tailwindcss.com/docs/space), or the [Gap utilities](https://tailwindcss.com/docs/gap) for flex or grid layout.

**Reply by verbose117** [reply]
oops, sorry! I didn't realise
